account_offbalance_sponsorship/models/account_move.py

```diff
@@ -0,0 +1,115 @@
+from odoo import models, api
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    def js_remove_outstanding_partial(self, partial_id):
+        """Called by the 'payment' widget to remove a reconciled entry from the present invoice.
+
+        :param partial_id: The id of an existing partial reconciliation linked to the current invoice.
+        """
+
+        self.ensure_one()
+        partial = self.env["account.partial.reconcile"].browse(partial_id)
+
+        if not partial:
+            return False
+
+        # Unreconcile
+        res = super().js_remove_outstanding_partial(partial_id)
+
+        # Get off-balance accounts
+        off_rec, off_ass = self.line_ids.get_account_offbalance(self.company_id)
+        account_32110 = self.env["account.account"].search([
+            ("code", "=", "32110"),
+            ("company_id", "=", self.company_id.id),
+        ], limit=1).id
```

Comment:
Was it like this in v14? I wonder if we should either:

1. Move the module in compassion-nordic, because it's specific to them and only meant to be used by them
2. Implement a configurable approach to find the desired account without hardcoding the account code (would it be either `account_offbalance_receivable` or `account_offbalance_asset` added in the settings)?

@davidwul @robinrolle What do you think?